collections/app/store/CollectionsStore.scala

```diff
@@ -27,7 +27,7 @@ object CollectionsStore {
   }
 
   def remove(collectionPath: List[String]): Future[Unit] = {
-    val path = CollectionsManager.pathToString(collectionPath).toLowerCase
+    val path = CollectionsManager.pathToLowerCaseString(collectionPath)
```

Comment:
Sorry to nitpick, but it could be `pathToPathId`.
It could then be [used here](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/model/Collection.scala#L11) too.